src/probnum/filtsmooth/_kalman_filter_smoother.py

```diff
@@ -0,0 +1,192 @@
+"""Convenience functions for filtering and smoothing."""
+
+import numpy as np
+
+from probnum import problems, randprocs, randvars, statespace
+from probnum.filtsmooth import gaussian
+
+__all__ = ["kalman_filter", "rauch_tung_striebel_smoother"]
+
+
+def kalman_filter(
```

Comment:
```suggestion
def filter_kalman(
```

I think it is better to name a function after what it does (i.e. using a verb if possible). It does return a result, not a Kalman filter.

Reply:
I agree. I dont particularly like "filter_kalman" though... how about `filter(..., method="kalman")`? This would allow extending this interface to e.g. ensemble kalman filters or the like. I think the fact that this redefines a built-in, is not *the* worst thing in the world, as one would probably use it as `filtsmooth.filter(...)`. Any thoughts in this direction? A potential extension of this convenience function to nonlinear system might motivate naming as in `filter_linear_system(..., method="kalman")`?

Reply:
Agreement: `filter_kalman`